app/views/events/_my_schedule.html.erb

```diff
@@ -68,52 +71,58 @@
       </tr>
       </thead>
       <tbody>
-      <% sessions.each do |session| %>
-        <% selection = table_sessions[session] %>
-        <% unless selection.blank? %>
-          <tr>
-            <td><%= session.timeslot %></td>
-            <td><%= selection.table.scenario.long_name %></td>
-            <td>
-              <% if selection.is_a? RegistrationTable %>
-                Player
-                <% if selection.table.premium %>
-                  <%= button_to 'Return!', [@event, session, selection.table, selection], data: { confirm: "Are you sure you want to return your ticket for #{selection.table.long_name}? Returning your ticket will not refund your money." }, :class => 'btn btn-xs btn-danger', method: :delete %>
-                <% else %>
-                  <%= button_to 'Return!', [@event, session, selection.table, selection], data: { confirm: "Are you sure you want to return your ticket for #{selection.table.long_name}?" }, :class => 'btn btn-xs btn-danger', method: :delete %>
-                <% end %>
-              <% elsif selection.is_a? GameMaster %>
-                GM
-              <% end %>
-            </td>
-            <td>
-              <%# handle online differently %>
-              <%= (selection.is_a? GameMaster) ? selection.table.location : 'See Musterer' %>
-            </td>
-            <td>
-              <div style="display: inline-block"><%= link_to 'View Table Details', event_session_table_path(event, session, selection.table) %></div>
-            </td>
-            <% if user_event.gamemaster? %>
+      <% if !hasSession %>
+        <tr>
+          <td>You are not signed up for any sessions!</td>
+        </tr>
+      <% else %>
```

Comment:
This is the main change. I thought the rest of this was spacing but maybe I'm missing a change from this main branch...

Reply:
The rest was whitespace.
I really need to get rubocop in place, so that we can once and for all have a linter enforcing code style